src/add/operator/combineLatest.ts

```diff
@@ -5,6 +5,6 @@
 import {Observable} from '../../Observable';
 import {combineLatest} from '../../operator/combineLatest';
 
-Observable.prototype.combineLatest = combineLatest;
+Observable.prototype.combineLatest = <any>combineLatest;
```

Comment:
yes, because `combineLatest` (the method) no longer has the same definition as `combineLatest` the prototype method.

I plan to update the script to add this in, and that's when I found out that the generation script didn't work the other day.